README.md

```diff
@@ -46,4 +37,63 @@ If you are looking to work on unit tests for this project be sure to read the RE
 
 `npx hardhat test` - run the unit tests
 
-<p align="center"><img src="https://media.giphy.com/media/rgwNTGFUbNTgsgiYha/giphy.gif"></p>
+----
+## Contracts 
+### **Baal (contracts/Baal.sol)**
+is a minimal yet composable DAO template continuing work from
+the Moloch, Minion and Compound frameworks to make it easier for
+people to combine and command crypto assets with intuitive membership
+games.
+
+#### Interfaces
+Minimal Clone Factoy [EIP 1167](https://eips.ethereum.org/EIPS/eip-1167)
+Gnosis Safe Module [Zodiac](https://github.com/gnosis/zodiac)
+
+### **Shares (contracts/SharesERC20.sol)**
+have direct execution, voting and exit rights around actions
```

Comment:
have direct execution, voting, and exit rights  ------ needs the extra comma after voting